### PR TITLE
Add Niskanen Center logo to US homepage

### LINKED ISSUE #1611: Add Niskanen Center logo to US homepage
Let's move it to the center (where the Maryland Child Alliance logo is), and move the Maryland Child Alliance logo to replace End Poverty Make Trillions

### PR DESCRIPTION
**Pull request by shaunakwagh**
## Description

Fixed #1611

## Changes

Added Niskanen Center Logo and moved the Maryland Child Alliance logo to replace End Poverty Make Trillions

## Screenshots

Screenshot 2024-04-16 at 8 10 23 PM

### CODE REVIEW

**Comment by MaxGhenis** [comment]
Please use this instead

image

File: src/images/logos/orgs/niskanen_center_logo.png
Comment (MaxGhenis) [inline src/images/logos/orgs/niskanen_center_logo.png:1]: rename niskanen-center.png for consistency with others

File: src/redesign/data/Organisations.jsx [hunk 1]
```diff
@@ -81,21 +82,22 @@ export const orgData = {
...
       link: "https://garycommunity.org",
     },
-    mca: {
-      name: "Maryland Child Alliance",
-      logo: MCA,
-      link: "https://www.marylandchildalliance.org/revenue-raisers",
+    niskanen_center: {
+      name: "Niskanen Center",
+      logo: NISKANENCenter,
+      link: "https://www.niskanencenter.org/",
```
Comment (MaxGhenis) [inline src/redesign/data/Organisations.jsx:98]:
```suggestion
      link: "https://www.niskanencenter.org/building-a-stronger-foundation-for-american-families-options-for-child-tax-credit-reform/",
```

Comment (shaunakwagh) [reply src/redesign/data/Organisations.jsx:98]: Ok thank you

**Comment by MaxGhenis** [comment]
thanks can you drop a screenshot?

**Comment by shaunakwagh** [comment]
Screenshot 2024-04-16 at 10 50 25 PM

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**